src/types/user.type.js

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+
+const GeoLocalisationType = PropTypes.shape({
+    lat: PropTypes.string,
+    lng: PropTypes.string,
+});
+
+const AddressType = PropTypes.shape({
```

Comment:
Where `AddressType` is used? Does it have `defaultProps`?

Reply:
`UserType` was getting a bit big and I thought of splitting it into small types, so in the future, we could use these types in some other components.
The suggested structure proposes that:

- the `UserType` has an `AddressType` and a `CompanyType`, and
- the `AddressType` has a `GeoLocalisationType`.

These types could be used later for more custom component that displays users address, company and geolocalisation.

I would say that we can have them as required in [line 27](https://github.com/MCRcodes/blog-demo/pull/3/files#diff-b0527e1ca8de25517f862fd3b1647903186bb5ec21741d417d1e6e2b83918046R27), [30](https://github.com/MCRcodes/blog-demo/pull/3/files#diff-b0527e1ca8de25517f862fd3b1647903186bb5ec21741d417d1e6e2b83918046R30) and [13](https://github.com/MCRcodes/blog-demo/pull/3/files#diff-b0527e1ca8de25517f862fd3b1647903186bb5ec21741d417d1e6e2b83918046R13), since according to the data we're consuming from the API, this information is always available.